components/playlist/resources/api/api.ts

```diff
@@ -32,16 +35,22 @@ class PlaylistAPIInstance implements API {
         this.pageHandler.$.bindNewPipeAndPassReceiver())
   }
 
-  getAllPlaylists = async function ():
-      Promise<{playlists: PlaylistMojo.Playlist[]}> {
-        return this.pageHandler.getAllPlaylists()
-      }
+  async getAllPlaylists () {
```

Comment:
I'm curious about whether we need this API interface - the PlaylistAPIInstance appears to specify only the stuff on the interface. I suspect removing it would be fine

Reply:
Sorry, I'm not sure if I understand what you're suggesting. Do you mean that removing getAllPlaylists from `API` interface and `PlaylistAPIInstance`? And make all callsites use `GetPlaylistAPI().pageHandler.getAllPlaylists()`? Or something else?

Reply:
Ah sorry, I meant we can probably delete the `API` interface - it doesn't seem to be used for anything we couldn't use `PlaylistAPIInstance` for.

Reply:
Oh, yes. I prefer private member too. Done!